monero-wallet-gui.pro

```diff
@@ -257,16 +257,21 @@ macx {
 # translation stuff
 TRANSLATIONS =  \ # English is default language, no explicit translation file
                 $$PWD/translations/monero-core_ar.ts \ # Arabic
-                $$PWD/translations/monero-core_de.ts \ # Deutsch
+                $$PWD/translations/monero-core_br.ts \ # Portuguese (Brazil)
+                $$PWD/translations/monero-core_de.ts \ # German
+                $$PWD/translations/monero-core_eo.ts \ # Esperanto
                 $$PWD/translations/monero-core_es.ts \ # Spanish
+                $$PWD/translations/monero-core_fi.ts \ # Finnish
                 $$PWD/translations/monero-core_fr.ts \ # French
                 $$PWD/translations/monero-core_hr.ts \ # Croatian
+                $$PWD/translations/monero-core_id.ts \ # Indonesian
                 $$PWD/translations/monero-core_in.ts \ # Hindi
                 $$PWD/translations/monero-core_it.ts \ # Italian
                 $$PWD/translations/monero-core_ja.ts \ # Japanese
+                $$PWD/translations/monero-core_nl.ts \ # Dutch
                 $$PWD/translations/monero-core_pl.ts \ # Polish
                 $$PWD/translations/monero-core_ru.ts \ # Russian
-                $$PWD/translations/monero-core_zh.ts \ # Chineese
+                $$PWD/translations/monero-core_zh.ts \ # Chinese (Taiwan)
```

Comment:
I would strongly recommend you call this 'traditional' rather than naming Taiwan specifically. Real political difficulties lie in that direction.

Traditional characters are used in Hong Kong, Macau and Taiwan, and Chinese diaspora in various other parts of the world.